### PR TITLE
changed to use moment-duration-format

### PR DESCRIPTION
**Pull request by alex-cory**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,7 +16,10 @@ ReactDOM.render(
 if (module.hot) {
   module.hot.accept('./appProvider.js', () => {
     const NextAppProvider = AppProvider.default;
-    ReactDOM.render(<NextAppProvider />, document.getElementById('root'));
+    render(
+      <NextAppProvider />,
+      document.getElementById('root')
```
Comment (dwalintukan) [inline src/index.js]: this is for refreshing right?

File: src/scenes/Event/scenes/oracle.js [hunk 2]
```diff
@@ -690,6 +689,7 @@ export default class OraclePage extends React.Component {
     const { oracle } = this.state;
     const totalAmount = _.sum(oracle.amounts);
     const { locale, messages: localeMessages } = this.props.intl;
+    window.oracle = oracle;
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: is this normal to have the window hold the var?

File: src/index.js (hunk 1 above)
Comment (alex-cory) [reply src/index.js]: honestly, I don't think we actually need this. I rewrote all this in my previous company and I removed this and refreshing worked just fine.

File: src/scenes/Event/scenes/oracle.js (hunk 2 above)
Comment (alex-cory) [reply src/scenes/Event/scenes/oracle.js]: ahhh! Sorry. Totally debugging and forgot to remove. Good catch

File: src/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/index.js]: lets leave this to the cleanup after mainnet release. for now, if its not broke, dont fix it.

**Comment by dwalintukan** [comment]
:shipit:

**Pull request merged**